packages/big-design/src/components/Message/styled.ts

```diff
@@ -44,7 +45,23 @@ export const StyledLink = styled(Link)`
   vertical-align: middle;
 `;
 
+export const StyledActionsWrapper = styled(Flex)`
+  flex-direction: row;
+  flex-wrap: wrap;
+  margin-bottom: -${({ theme }) => theme.spacing.xSmall};
+  margin-left: -${({ theme }) => theme.spacing.xxSmall};
+  margin-right: -${({ theme }) => theme.spacing.xxSmall};
+`;
+
+export const StyledActionWrapper = styled(FlexItem)`
+  margin-bottom: ${({ theme }) => theme.spacing.xSmall};
+  margin-left: ${({ theme }) => theme.spacing.xxSmall};
+  margin-right: ${({ theme }) => theme.spacing.xxSmall};
+`;
```

Comment:
Why not get rid of the `ActionWrapper` and let the `Buttons` be the flex items, they have the margins already:

```tsx
<Button
  key={index}
  variant={getActionVariant(variant)}
  {...excludeMarginProps(actionProps)}
  mobileWidth="auto"
>
```

Note: I included the `excludeMarginProps` utility so that users can't add their own margins. You might want to enforce this on the type too.